src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -38,8 +38,10 @@
 					:disabled="isSaving"
 					class="password-form__input-field"
 					label-visible
-					:label="t('spreed', 'Enter new password')" />
-				<NcButton :disabled="isSaving" type="primary" native-type="submit">
+					:label="t('spreed', 'Enter new password')"
+					@valid="isValid = true"
+					@invalid="isValid = false" />
+				<NcButton :disabled="isSaving || !isValid" type="primary" native-type="submit">
```

Comment:
Something off with styles (locally and on video)
Nevertheless, align to top (considering margin) and flex-shrink: 0 for the button, to not squeeze it?

![image](https://github.com/user-attachments/assets/83450ec8-e7f5-4970-97ce-dbb20bf6e79d)